packages/gcm/lib/apk/config.yaml

```diff
@@ -0,0 +1,11 @@
+registration-class: org.domokit.gcm.RegistrationIntentService$MojoService
+jars:
```

Comment:
These blobs need to go somewhere else.  GCM is not open source and cannot land in this open-source repository.